tasks/python-reference-activity/README.md

```diff
@@ -0,0 +1,32 @@
+# @cumulus/python-reference-activity
+
+This 'task' is a reference activity implementation that is included with Cumulus to allow integration testing of the `Cumulus message adapter` and [`cumulus-process-py`](https://github.com/nasa/cumulus-process-py) with a deployed activity.
+
+## Development
+
+Developmental use of this lambda is intended to be simple - the processing activity runs an activity through the CMA and returns a static processing output, integration tests can be then built against the `Reference Activity` step in the `PythonProcess` workflow.
```

Comment:
Nit: I would link `PythonProcess` here as well. I would also use this link style

```
[I'm a reference-style link][Arbitrary case-insensitive reference text]

... 

[arbitrary case-insensitive reference text]: https://www.mozilla.org
```

That way you can define the target for links that repeat throughout the document

Reply:
f416c8ed8